website/docs/r/kusto_cluster.html.markdown

```diff
@@ -70,6 +70,12 @@ The following arguments are supported:
 
 * `virtual_network_configuration` - (Optional) A `virtual_network_configuration` block as defined below. Changing this forces a new resource to be created.
 
+* `virtual_network_configuration_enabled` - (Optional) Whether the virtual network is enabled. This property could only be set when `virtual_network_configuration` is present. Possible values are: `true` and `false`. Defaults to `true`.
+
+~> **NOTE:** Currently `virtual_network_configuration_enabled` could only change from `true` to `false` and not the other way around.
+
+~> **NOTE:** Changing `virtual_network_configuration_enabled` to `false` will cause the `public_network_access_enabled` field to be set to `false` as well.
+
```

Comment:
```suggestion
* `virtual_network_configuration_enabled` - (Optional) Specifies if the `virtual_network_configuration` code block settings are `enabled` or not. The `virtual_network_configuration_enabled` field can only be defined if the `virtual_network_configuration` code block is present in the configuration file. Can only be set to `true`. Defaults to `false`.

~> **NOTE:** To `disable` the `virtual_network_configuration` code block you must remove the  `virtual_network_configuration_enabled` field from your configuration file. Once the `virtual_network_configuration_enabled` field has been removed it cannot be readded.

~> **NOTE:** By removing the `virtual_network_configuration_enabled` field from your configuration file this will also toggle the `public_network_access_enabled` field to be set to `false` as well.

```